docs/specification/2025-03-26/basic/authorization.mdx

```diff
@@ -314,95 +494,91 @@ Servers **MUST** return appropriate HTTP status codes for authorization errors:
 | 403         | Forbidden    | Invalid scopes or insufficient permissions |
 | 400         | Bad Request  | Malformed authorization request            |
 
-### 2.9 Implementation Requirements
-
-1. Implementations **MUST** follow OAuth 2.1 security best practices
-2. PKCE is **REQUIRED** for all clients
-3. Token rotation **SHOULD** be implemented for enhanced security
-4. Token lifetimes **SHOULD** be limited based on security requirements
+### 2.12 Implementation Requirements
 
-### 2.10 Third-Party Authorization Flow
+1. Implementations **MUST** follow OAuth 2.1 security best practices.  
+2. PKCE is **REQUIRED** for all clients.  
```

Comment:
This bullet point is already covered by point 1, given that PKCE is required for all clients using the authorization code flow in OAuth 2.1.